integration_test/registration_test.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:flutter_test/flutter_test.dart';
 import 'package:flutter/material.dart';
 import 'package:firebase_auth/firebase_auth.dart';
-//import 'package:integration_test/integration_test.dart';
+// import 'package:integration_test/integration_test.dart';
```

Comment:
Why is there this commented out import statement here ? If no reason, it's better to remove it

Reply:
This isn't my code nor file but I will remove it anyway - thanks!